stl/inc/xstring

```diff
@@ -2744,8 +2730,11 @@ private:
         _Container_proxy_ptr<_Alty> _Proxy(_Alproxy, _My_data);
 
         if (_Count < _BUF_SIZE) {
+            _My_data._Myres = _BUF_SIZE - 1;
+
             _My_data._Mysize = _Count;
-            _My_data._Myres  = _BUF_SIZE - 1;

```

Comment:
Is this just churn?

Reply:
I wanted to put the `size` set next to the `_ASAN_STRING_CREATE`